graphene_django/tests/test_query.py

```diff
@@ -996,14 +947,23 @@ class Query(graphene.ObjectType):
         }
     """
 
+    def str_to_node_id(val):
+        return base64.b64encode(val.encode()).decode()
```

Comment:
I think we should use `graphql_relay.to_global_id` here.

Reply:
:thumbsup: done